src/CONST.js

```diff
@@ -279,6 +279,9 @@ const CONST = {
     CFPB_PREPAID_URL: 'https://cfpb.gov/prepaid',
     STAGING_NEW_EXPENSIFY_URL: 'https://staging.new.expensify.com',
     NEWHELP_URL: 'https://help.expensify.com',
+    DEV_EXPENSIFY_URL: 'https://www.expensify.com.dev',
```

Comment:
IIRC `DEV_EXPENSIFY_URL` is the same as PROD

This address doesn't work: https://www.expensify.com.dev

Reply:
You need to have a local VM for this to work so this works only for internal developers

Reply:
Yes, dev is different from prod. It links to internal engineer's VM.

Reply:
Let's update it to Internal as contributors dev use Prod URL. 👍